### ISSUE TITLE
CDK depends on vulnerable version of netmask (CVE-2021-28918)

### ISSUE THREAD

**Issue by mlafeldt** [aws/aws-cdk#13873]
We noticed this today thanks to Dependabot.

```
CVE-2021-28918
high severity
Vulnerable versions: < 2.0.1
Patched version: 2.0.1
netmask npm package is vulnerable to octal input data. This may lead to server-side request forgery, remote file inclusion, local file inclusion, and other vulnerabilities.

See the researcher report for an extensive write-up.
```

See https://sick.codes/universal-netmask-npm-package-used-by-270000-projects-vulnerable-to-octal-input-data-server-side-request-forgery-remote-file-inclusion-local-file-inclusion-and-more-cve-2021-28918/ for details.

Here's the dependency graph:

```
$ yarn why netmask
yarn why v1.22.10
[1/4] 🤔  Why do we have the module "netmask"...?
[2/4] 🚚  Initialising dependency graph...
[3/4] 🔍  Finding dependency...
[4/4] 🚡  Calculating file sizes...
=> Found "netmask@1.0.6"
info Reasons this module exists
   - "_project_#infrastructure#aws-cdk#proxy-agent#pac-proxy-agent#pac-resolver" depends on it
   - Hoisted from "_project_#infrastructure#aws-cdk#proxy-agent#pac-proxy-agent#pac-resolver#netmask"
info Disk size without dependencies: "60KB"
info Disk size with unique dependencies: "60KB"
info Disk size with transitive dependencies: "60KB"
info Number of shared dependencies: 0
✨  Done in 0.70s.
```

**Comment by peterwoodworth** [comment]
Thank you very much for the concern and report! We have identified and applied a fix that will upgrade npm users to netmask@2 as of the next release.

**Comment by mlafeldt** [comment]
Fixed by #13914 and released as part of [1.95.2](https://github.com/aws/aws-cdk/blob/master/CHANGELOG.md#1952-2021-04-01).
